src/index.css

```diff
@@ -1,13 +1,27 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
```

Comment:
- [ ] As per the given design guidelines, please use the given font family so as to try and match the given design as much as possible.
![image](https://github.com/malikhaiderkhan/Air-pollution/assets/81932478/3c15fa85-876c-468d-9704-ec7d590fd523)